src/Sarif.Viewer.VisualStudio.Core/Tags/SarifLocationErrorTagger.cs

```diff
@@ -98,10 +97,10 @@ public IEnumerable<ITagSpan<IErrorTag>> GetTags(NormalizedSnapshotSpanCollection
                 yield break;
             }
 
+            var groupedBySpan = new Dictionary<(int start, int end), (List<IErrorTag> tagList, SnapshotSpan snapshotSpan)>();
```

Comment:
How do the changes in ErrorTagger impact the log in/out functionality?

Reply:
Not sure why github is showing these changes here at all, they were made in [this](https://github.com/microsoft/sarif-visualstudio-extension/pull/603/files#diff-9fcab188774650e3ac62f050b43e04c1e3bbac6920dc3701cd427ecd673975dd) pr that was merged into main and then merged into the feature branch [here](https://github.com/microsoft/sarif-visualstudio-extension/commit/654e7ad071cd302d74ccad14b53dfad3b1ab8bee)